tests/e2e_test/kyma_deletion_test.go

```diff
@@ -12,71 +12,84 @@ import (
 var _ = Describe("KCP Kyma CR Deletion", Ordered, func() {
 	kyma := NewKymaWithSyncLabel("kyma-sample", "kcp-system", "regular",
 		v1beta2.SyncStrategyLocalSecret)
+	module := NewTestModuleWithFixName(moduleName, v1beta2.DefaultChannel)
```

Comment:
Please use `NewTemplateOperator` to create a template-operator module, it's in `testutils`